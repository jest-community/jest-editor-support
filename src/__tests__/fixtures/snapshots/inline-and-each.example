describe('tests with each', () => {
  it.each`
  case|whatever
  ${1}|${'a'}
  $(2)|${'b'}
  `('case $case: test tabled each', ({whatever}) => {
      expect(whatever).toMatchSnapshot();
      expect(whatever).toMatchInlineSnapshot();
  });
  it.each([1,2,3])('case %d test 1-D array each', (n) => {
      expect(n).toThrowErrorMatchingSnapshot();
      expect(n).toMatchInlineSnapshot();

  });
});

describe.each([1,2,3])('%d describe with each', (n) => {
  it.each(['a', 'b', 'c'])('test.each %s', (char) => {
      expect({n, char}).toMatchSnapshot();
  });
  it('a regular test', () => {
    expect(n).toMatchInlineSnapshot();
  });
});

it.each(['a', 'b', 'c'])('inline test.each %s', (char) => {
  expect(char).toThrowErrorMatchingInlineSnapshot();
});
it.each(['a', 'b', 'c'])('test.each %s', (char) => {
  expect(char).toMatchSnapshot();
});
it('regular inline test', () => {
  expect(whatever).toMatchInlineSnapshot();
});

