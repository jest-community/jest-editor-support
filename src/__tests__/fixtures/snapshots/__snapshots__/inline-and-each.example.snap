// Jest Snapshot v1, https://goo.gl/fbAQLP
exports[`test.each a 1`] = `a`;
exports[`test.each b 1`] = `b`;
exports[`test.each c 1`] = `c`;
exports[`1 describe with each test.each a 1`] = `1.a`;
exports[`1 describe with each test.each b 1`] = `1.b`;
exports[`1 describe with each test.each c 1`] = `1.c`;
exports[`2 describe with each test.each a 1`] = `2.a`;
exports[`2 describe with each test.each b 1`] = `2.b`;
exports[`2 describe with each test.each c 1`] = `2.c`;
exports[`3 describe with each test.each a 1`] = `3.a`;
exports[`3 describe with each test.each b 1`] = `3.b`;
exports[`3 describe with each test.each c 1`] = `3.c`;
exports[`tests with each case 1 test 1-D array each 1`] = `1 1-D`;
exports[`tests with each case 2 test 1-D array each 1`] = `2 1-D`;
exports[`tests with each case 3 test 1-D array each 1`] = `3 1-D`;
exports[`literal test 1`] = `literal test 1 content`;
exports[`literal test 2`] = `literal test 2 content`;
